configs/example/gem5_library/x86-npb-benchmarks.py

```diff
@@ -98,15 +94,32 @@
     type=str,
     required=True,
     help="Input the benchmark program to execute.",
-    choices=benchmark_choices,
-)
-
-parser.add_argument(
-    "--size",
-    type=str,
-    required=True,
-    help="Input the class of the program to simulate.",
-    choices=size_choices,
+    choices=[
+        "npb-bt-A",
+        "npb-bt-B",
+        "npb-bt-C",
+        "npb-cg-A",
+        "npb-cg-B",
+        "npb-cg-C",
+        "npb-ep-A",
+        "npb-ep-B",
+        "npb-ep-C",
+        "npb-ft-A",
+        "npb-ft-B",
+        "npb-ft-C",
+        "npb-is-A",
+        "npb-is-B",
+        "npb-is-C",
+        "npb-lu-A",
+        "npb-lu-B",
+        "npb-lu-C",
+        "npb-mg-A",
+        "npb-mg-B",
+        "npb-mg-C",
+        "npb-sp-A",
+        "npb-sp-B",
+        "npb-sp-C",
+    ],
```

Comment:
Can we get this from the suite instead of writing them all out here?

Reply:
We can, I did this at the start, but when we initialize the suite it downloads all the resources that are required so it will also download everything when we do `-h` to get benchmark choices.

Reply:
That sounds like a bug/missing feature. If I run obtain_resource("x86-npb-suite") it should really just download the information needed for the suite and lazily download the data needed for workloads.

Reply:
I will push a patch that would download the workload and its dependent resources when we first call them in the iterator. I will also add a function that just returns the metadata for a workload only.

Reply:
PR #609 has changes to lazily download workloads.